docs/subnets/how-to-import-public.md

```diff
@@ -0,0 +1,208 @@
+
+The most probable reason why someone would want to do this, 
+is if they already deployed a Subnet with `subnet-cli` either to `Fuji` or `Mainnet`, 
+and now would like to use `Avalanche-CLI` to manage the Subnet.  
+
+`subnet-cli` is deprecated.
+
+Similarly, a Subnet might have been created "manually" by issuing transactions 
+to node APIs (either a local node or public API nodes) with no help of any tool so far,
+but would now require `Avalanche-CLI` integration.
+
+For this How-To, we will import the `wagmi` Subnet from `Fuji`.
+
+
+## Requirements
+
+For the import to work properly: 
+
+* the *genesis* for the Subnet in question should be known and accessible as a file on disk.
+
+* the Subnet's *ID* should be known, and needs to be supplied to the wizard when prompted
+
+
+## Import the Subnet
+
+For these use cases, `Avalanche-CLI` now supports the `import public` command.
+
+Start the import by issuing
+
+```shell
+avalanche subnet import public 
+```
+
+The tool will prompt for the network from which to import. 
+The invariant assumption here is that the network is a public network 
+(in other words, importing from a local network is not supported).
+
+```shell
+Use the arrow keys to navigate: ↓ ↑ → ← 
+? Choose a network to import from: 
+  ▸ Fuji
+    Mainnet
+```
+
+As stated earlier, we will import from `Fuji`, so select it. 
+As a next step, `Avalanche-CLI` asks for the path of the genesis file on disk:
+
+```shell
+✗ Provide the path to the genesis file: /tmp/subnet_evm.genesis.json
+```
+
+It will be checked if the file at the provided path exists 
+(refer to check mark at the beginning of the line):
+
+```shell
+✔ Provide the path to the genesis file: /tmp/subnetevm_█enesis.json
+```
+
+Subsequently, the wizard asks if nodes have already been deployed for this Subnet. 
+
+```shell
+Use the arrow keys to navigate: ↓ ↑ → ← 
+? Have nodes already been deployed to this subnet?: 
+    Yes
+  ▸ No
+```
+
+### Nodes are Already Validating This Subnet
+
+If they have been, it will attempt to query such a node for detailed data like the VM version. 
+This will allow to skip having to query github (or wherever the VM's repository is hosted) 
+for the VM's version, but rather we'll get the exact version which is actually running on the node.
+
+For this to work, an node API URL is requested from the user, which will be used for the query.
+This will require that the node's API IP and port are accessible from the machine running 
+`Avalanche-CLI`, or the node will obviously not be reachable, 
+and thus the query will timeout and fail, and the tool exit. 
+The node should also be validating the given Subnet for the import to be meaningful, 
+otherwise, the import will fail with missing information.
+
+If the query succeeded, the wizard jumps to prompt for the Subnet ID.
+
+```shell
+Please provide an API URL of such a node so we can query its VM version (e.g. http://111.22.33.44:5555): http://154.42.240.119:9650
```

Comment:
Is this IP address an actual IP address? If so should we remove it or replace it w/ an example IP? http://154.42.240.119:9650

Reply:
I actually took a real IP from our nodes with which I tested, and then altered it (in other words: no, it is not an actual address). 

Should I add a note saying it is not a real IP?

Reply:
No I think it's fine as it is in that case.